tests/system/python/conftest.py

```diff
@@ -215,6 +215,25 @@ def clone_make_install():
 
     return _add_fledge_north
 
+
+@pytest.fixture
+def update_stat_collection(fledge_url, wait_time):
+    """Update the Stat colectioin of all south service to per asset & service"""
```

Comment:
`Statistics Collection`

Also this fixture should provide facility for a single south service as well.